src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const log = [];
+
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
you can destructure keys from object

Reply:
➕ , you can destructure all the needed properties